src/unix/bsd/freebsdlike/freebsd.rs

```diff
@@ -4,7 +4,16 @@ pub type fsfilcnt_t = ::uint64_t;
 pub const PTHREAD_STACK_MIN: ::size_t = 2048;
 pub const KERN_PROC_PATHNAME: ::c_int = 12;
 pub const SIGSTKSZ: ::size_t = 34816;
+pub const SF_NODISKIO: ::c_int = 0x00000001;
+pub const SF_MNOWAIT: ::c_int = 0x00000002;
+pub const SF_SYNC: ::c_int = 0x00000004;
+pub const SF_NOCACHE: ::c_int =   0x00000010;
 
 extern {
     pub fn __error() -> *mut ::c_int;
 }
+
+#[inline]
+pub fn SF_FLAGS(rh: ::c_int, flags: ::c_int) -> ::c_int {
```

Comment:
It's actually brand new in FreeBSD: https://svnweb.freebsd.org/base?view=revision&revision=293439

I imagine it's not absolutely needed, as the man pages document the use of the upper and lower 16 bits for `flags`. I've removed it for now.